src/common/eeprom_current.hpp

```diff
@@ -31,11 +31,12 @@ struct vars_body_t : public eeprom::v10::vars_body_t {
     char WIFI_HOSTNAME[LAN_HOSTNAME_MAX_LEN + 1];
     char WIFI_AP_SSID[WIFI_MAX_SSID_LEN + 1];
     char WIFI_AP_PASSWD[WIFI_MAX_PASSWD_LEN + 1];
+    uint8_t USB_MSC_ENABLED;
 };
 
 #pragma pack(pop)
 
-static_assert(sizeof(vars_body_t) == sizeof(eeprom::v10::vars_body_t) + sizeof(uint8_t) * 3 + PL_API_KEY_SIZE + LAN_HOSTNAME_MAX_LEN + 1 + WIFI_MAX_SSID_LEN + 1 + WIFI_MAX_PASSWD_LEN + 1 + sizeof(uint32_t) * 5, "eeprom body size does not match");
+static_assert(sizeof(vars_body_t) == sizeof(eeprom::v10::vars_body_t) + sizeof(uint8_t) * 3 + PL_API_KEY_SIZE + LAN_HOSTNAME_MAX_LEN + 1 + WIFI_MAX_SSID_LEN + 1 + WIFI_MAX_PASSWD_LEN + 1 + 1 + sizeof(uint32_t) * 5, "eeprom body size does not match");
```

Comment:
Well, this somehow checks whether the eeprom's body size is correct. I've added one one-byte variable, so I had to „add one“ here. 

I am not the author of this assert. I am just using it the way (I believe) it was designed to be used :)

Reply:
I am not sure if you don't need to create new eeprom version instead of adding something to current version. Or eeprom_current means not released yet?

Reply:
I believe it is the unreleased one.

Reply:
ok